rpc-protocol/src/main/java/org/sonarsource/sonarlint/core/rpc/protocol/backend/initialize/FeatureFlagsDto.java

```diff
@@ -34,10 +34,11 @@ public class FeatureFlagsDto {
   private final boolean shouldManageServerSentEvents;
   private final boolean enableDataflowBugDetection;
   private final boolean shouldManageFullSynchronization;
+  private final boolean enableTelemetry;
 
 
   public FeatureFlagsDto(boolean shouldManageSmartNotifications, boolean taintVulnerabilitiesEnabled, boolean shouldSynchronizeProjects, boolean shouldManageLocalServer,
-    boolean enableSecurityHotspots, boolean shouldManageServerSentEvents, boolean enableDataflowBugDetection, boolean shouldManageFullSynchronization) {
+    boolean enableSecurityHotspots, boolean shouldManageServerSentEvents, boolean enableDataflowBugDetection, boolean shouldManageFullSynchronization, boolean enableTelemetry) {
```

Comment:
What do you think about creating a default constructor for an instance with default values? Using a builder would even be better, actually.
So we don't need to worry about the usage of IDEs.